docs/examples/kops-test-route53-subdomain.md

```diff
@@ -0,0 +1,870 @@
+
+```bash
+I0906 09:42:09.399908   13538 executor.go:91] Tasks: 0 done / 75 total; 38 can run
+I0906 09:42:12.033675   13538 vfs_castore.go:422] Issuing new certificate: "master"
+I0906 09:42:12.310586   13538 vfs_castore.go:422] Issuing new certificate: "kube-scheduler"
+I0906 09:42:12.791469   13538 vfs_castore.go:422] Issuing new certificate: "kube-proxy"
+I0906 09:42:13.312675   13538 vfs_castore.go:422] Issuing new certificate: "kops"
+I0906 09:42:13.378500   13538 vfs_castore.go:422] Issuing new certificate: "kubelet"
+I0906 09:42:13.398070   13538 vfs_castore.go:422] Issuing new certificate: "kube-controller-manager"
+I0906 09:42:13.636134   13538 vfs_castore.go:422] Issuing new certificate: "kubecfg"
+I0906 09:42:14.684945   13538 executor.go:91] Tasks: 38 done / 75 total; 14 can run
+I0906 09:42:15.997588   13538 executor.go:91] Tasks: 52 done / 75 total; 19 can run
+I0906 09:42:17.855959   13538 launchconfiguration.go:327] waiting for IAM instance profile "masters.mycluster01.kopsclustertest.kopeio.org" to be ready
+I0906 09:42:17.932515   13538 launchconfiguration.go:327] waiting for IAM instance profile "nodes.mycluster01.kopsclustertest.kopeio.org" to be ready
+I0906 09:42:18.602180   13538 launchconfiguration.go:327] waiting for IAM instance profile "masters.mycluster01.kopsclustertest.kopeio.org" to be ready
+I0906 09:42:18.682038   13538 launchconfiguration.go:327] waiting for IAM instance profile "masters.mycluster01.kopsclustertest.kopeio.org" to be ready
+I0906 09:42:29.215995   13538 executor.go:91] Tasks: 71 done / 75 total; 4 can run
+I0906 09:42:30.073417   13538 executor.go:91] Tasks: 75 done / 75 total; 0 can run
+I0906 09:42:30.073471   13538 dns.go:152] Pre-creating DNS records
+I0906 09:42:32.403909   13538 update_cluster.go:247] Exporting kubecfg for cluster
+Kops has set your kubectl context to mycluster01.kopsclustertest.kopeio.org
+
+Cluster is starting.  It should be ready in a few minutes.
+
+Suggestions:
+ * validate cluster: kops validate cluster
+ * list nodes: kubectl get nodes --show-labels
+ * ssh to the master: ssh -i ~/.ssh/id_rsa admin@api.mycluster01.kopsclustertest.kopeio.org
+The admin user is specific to Debian. If not using Debian please use the appropriate user based on your OS.
+ * read about installing addons: https://github.com/kubernetes/kops/blob/master/docs/addons.md
+```
+
+Note that KOPS will create a DNS record for your API: api.mycluster01.kopsclustertest.kopeio.org. You can check this record with the following "dig" command:
+
+```bash
+dig +short api.mycluster01.kopsclustertest.kopeio.org A
+34.228.219.212
+34.206.72.126
+54.83.144.111
+```
+
+KOPS created a DNS round-robin resource record with all the public IP's assigned to the masters. Do you remember we specified 3 masters ?. Well, there are their IP's.
+
+After about 10~15 minutes (depending on how fast or how slow are amazon services during the cluster creation) you can check your cluster:
```

Comment:
5-8 min?

Reply:
Amazon sometimes is not that fast. Better be safe and say 10-15 minutes.